onyo/cli/get.py

```diff
@@ -96,12 +100,14 @@
         action=StoreSortOption,
         nargs='+',
         help=r"""
-            Sort output by **SORT-KEY** in ascending order.
+            Sort matches by **SORT-KEY** in ascending order.
             Can be given multiple times. Sorting by multiple **SORT-KEY** will be done in order
             (earlier given keys take precedence over subsequent keys).
             This can be intermixed with ``-s/--sort-descending``.
             Note, that if a **SORT-KEY** appears multiple times, the latest appearance will
-            overrule what ws specified before.
+            overrule what was specified before.
+            This sorts the matching assets rather than the output.
```

Comment:
Nit: There's a tendency to over-explain here. But the user doesn't need to know /why/. They just need to know the feature.

Simply:

> One can sort by keys that are not in the output.